pages/content/posts/london-meetup/index.md

```diff
@@ -0,0 +1,34 @@
+---
+author: "Florian Scholz"
+title: "Open Web Docs meetup with Mozilla’s MDN Team in London"
```

Comment:
Consider dropping "in London", just for brevity.

Reply:
```suggestion
title: "Open Web Docs meetup with Mozilla’s MDN Team"
```